src/encoded/types/experiment.py

```diff
@@ -188,6 +188,22 @@ def superseded_by(self, request, superseded_by):
         },
     }
 
+    target_matrix = {
+        'y': {
+            'group_by': [
+                'replicates.library.biosample.donor.organism.scientific_name',
+                'target.label',
+            ],
+            'label': 'Target',
+        },
+        'x': {
+            'group_by': ['biosample_ontology.classification', 'biosample_ontology.term_name'],
+            'label': 'Term Name',
+        },
+        'viewableTabs': ['Homo sapiens', 'Mus musculus'],
```

Comment:
'viewableTabs' -> 'viewable_tabs'

Reply:
Remove the variable as it was used in old code.